v0/destinations/lytics/data/LYTICSTrackConfig.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "destKey": "_e",
```

Comment:
@ruchiramoitra According to [this](https://github.com/rudderlabs/rudder-sdk-js/pull/144#discussion_r527897323) do you think the `destKey` should be `event`?

Reply:
not for track... that is the case for page views only.